app/components/ui/Section/CategorySection.tsx

```diff
@@ -21,51 +22,152 @@ interface FeaturedCategoriesSection {
 export interface FeaturedCategoriesData {
   header: string;
   subheader: string;
-  backgroundColor: string;
+  backgroundColor: BackgroundColorVariant;
   featuredCategoriesSection: FeaturedCategoriesSection[];
 }
 
-interface CategorySectionProps {
-  sectionData: FeaturedCategoriesData;
-}
-
-export const CategorySection = (props: CategorySectionProps) => {
-  const { sectionData } = props;
+export const CategorySection = () => {
+  // const { sectionData } = props;
 
-  if (!sectionData) {
-    return <div>Loading...</div>;
-  }
+  // if (!sectionData) {
+  //   return <div>Loading...</div>;
+  // }
 
-  const { header, subheader, backgroundColor, featuredCategoriesSection } =
-    sectionData;
-  const featuredCategories = featuredCategoriesSection[0].category;
+  // const { header, subheader, backgroundColor, featuredCategoriesSection } =
+  //   sectionData;
+  // const featuredCategories = featuredCategoriesSection[0].category;
 
   return (
     <HomePageSection
-      title={header}
-      description={subheader}
-      backgroundColor={backgroundColor}
+      title={tempCategoriesSection.header}
+      description={tempCategoriesSection.subheader}
+      backgroundColor="secondary"
     >
       <div className={styles.categorySection}>
         <div className={styles.categoryCards}>
-          {featuredCategories.map((category) => {
+          {hardcodeCategories.map((category) => {
```

Comment:
Ok React just broke my brain that `hardcodeCategories` is not throwing a `ReferenceError` in this scope since it's defined below. 🤯